ppr-ui/src/components/tables/RegistrationTable.vue

```diff
@@ -381,8 +426,10 @@ export default defineComponent({
       loadingPDF: '',
       overrideWidth: false,
       registrationTypes: [...RegistrationTypesStandard].slice(1),
+      mhrRegistrationTypes: [...MHRegistrationTypes].slice(1),
       showDatePicker: false,
       statusTypes: [...StatusTypes],
+      mhStatusTypes: [...mhStatusTypes],
```

Comment:
`[...mhStatusTypes]` is same a `mhStatusTypes`

Reply:
Made the change...

Reply:
I think you made a change that was not intended. I meant to say that you don't need to use the spread operator and then put it back into the array like so `[...mhStatusTypes].` You can use `mhStatusTypes` directly as it is an array already.

Reply:
Ok

Reply:
Made the change... Kept the Camel case names for consistency.